src/panels/MissionsPanel/util.ts

```diff
@@ -0,0 +1,42 @@
+import { isDateValid } from '@/redux/time/util';
+
+export function jumpToTime(
+  currentTime: number | undefined,
+  time: string,
+  luaApi: OpenSpace.openspace | null,
+  fadeTime: number = 1
+): void {
+  if (currentTime === undefined) {
+    return;
+  }
+
+  const utcDate = new Date(time);
+
+  if (!isDateValid(utcDate)) {
+    return;
+  }
+
+  const timeDiffInSeconds = Math.abs(currentTime - utcDate.valueOf()) / 1000;
```

Comment:
```suggestion
  const timeDiffInSeconds = Math.abs(currentTime - utcDate.valueOf()) / 1000.0;
```
? 

See previous comment? Would like to discuss if we want to use the same coding style for numbers here as we do in C++